Gemfile

```diff
@@ -2,9 +2,6 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'octicons', '~> 2.0'
```

Comment:
The dependency is already defined inside the gemspec, and we are loading these dependencies through the `gemspec` method call on line 3 here.